examples/gym_ex/rl_agent_v2.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This contains the rl agent class."""
+
+import numpy as np
+import random
+from typing import Any, Dict
+
+from examples.gym_ex.env import BanditNArmedRandom
```

Comment:
Please revert these imports to the previous commit. We don't want examples and gym_ex to be packages. E.g. `examples.gym_ex.env` > `env`